lesson-4/public/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+  <head>Giving name and comment which are to be saved into txt file</head>
+<body>
+  <form action='/ha'>
+    <input type="text" name="username" placeholder="name"></input>
```

Comment:
One word: SIMPLE. 
You don't need placeholder in this code. Don't use it again next time.